### PR TITLE
add docs for new frameworks

### PR DESCRIPTION
**Pull request by shivam-sharma7**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by shivam-sharma7** [comment]
@melloware don't merge it, I will update once #444 got merged

**Review by melloware — APPROVED** [review]

**Pull request merged**
